backend/apis/users/patch-user.sql

```diff
@@ -0,0 +1,12 @@
+UPDATE users SET accessLevel = ? WHERE id = ?;
+
+INSERT IGNORE INTO
+  userPermissions (userId, permission)
+  SELECT ?, 'immigration' WHERE ?
```

Comment:
This is a way of doing a conditional insert. We inject `WHERE false` when we don't want to insert